Donut_Shop.html

```diff
@@ -4,9 +4,22 @@
 </head>
 <body>
   <p id="donuts"></p>
+    <form id="textBoxes">
+        <label for="franchise">Location: </label>
+        <input id="franchise" name="franchise" type="text">
```

Comment:
You could use placeholder attribs with your input tags to help the user understand what type of data goes into each.